save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/BanView.kt

```diff
@@ -0,0 +1,67 @@
+/**
+ * View for Ban
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.views
+
+import com.saveourtool.save.frontend.utils.Style
+import com.saveourtool.save.frontend.utils.useBackground
+import com.saveourtool.save.info.UserInfo
+import com.saveourtool.save.info.UserStatus
+import js.core.jso
+import react.FC
+import react.PropsWithChildren
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.p
+import react.router.useNavigate
+import web.cssom.ClassName
+import web.cssom.rem
+
+private const val BAN = """
+    BAN
+"""
+
+private const val SUPPORT = """
+    You have received a ban on our platform. If you think that this happened by mistake, please contact our support.
+"""
+
+@JsExport
+@OptIn(ExperimentalJsExport::class)
+val banView: FC<BanProps> = FC { props ->
+    useBackground(Style.SAVE_LIGHT)
+
+    val navigate = useNavigate()
+    if (props.userInfo?.status != UserStatus.BANNED) {
+        navigate("/", jso { replace = true })
+    }
+
+    div {
```

Comment:
in bootstrap all wrapper divs should be cols or rows
Please follow this notation